Resources/Prototypes/_Goobstation/Admin_shit/droppods.yml

```
@@ -0,0 +1,59 @@
+- type: entity
+  parent: BaseItem
+  id: DropPodspawner
+  name: drop pod
+  description: Place items you want players to recieve in the bag, activate it over where you want the pod to drop. Players cannot see the bag. To drop living beings, give them the item component, place them in the bag, then remove the component.
+  components:
+  - type: Sprite
+    sprite: _Goobstation/Objects/Specific/DropPod/item.rsi
+    state: DROPPOD
+  - type: Item
+    size: Ginormous
+  - type: Storage
+    grid:
+    - 0,0,20,20
+    maxItemSize: Ginormous
+  - type: Ghost
+  - type: ContainerContainer
```

This won't need a ghost type, admins should just not have it on the floor. Causes integration test fails.
```suggestion
  - type: ContainerContainer
```